Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Error.swift

```diff
@@ -150,9 +150,17 @@ extension PathHierarchy.Error {
             let solutions: [Solution] = candidates
                 .sorted(by: collisionIsBefore)
                 .map { (node: PathHierarchy.Node, disambiguation: String) -> Solution in
-                    return Solution(summary: "\(Self.replacementOperationDescription(from: disambiguations.dropFirst(), to: disambiguation)) for\n\(fullNameOfNode(node).singleQuoted)", replacements: [
-                        Replacement(range: replacementRange, replacement: "-" + disambiguation)
-                    ])
+                    // Suggest removing the disambiguation hash entirely
+                    if disambiguation.isEmpty {
```

Comment:
Ideally this would happen automatically, without the special check here.

Reply:
I found that we need to keep this check. It is possible to remove this check entirely, but then we would have to add the `-` prefix to _all_ the unknownDisambiguation warnings throughout all the unit tests.

Example:

```
Replace '-abc123' with '-method' for\n'func `init`()'"
```

I think this reads fine for users, but I wouldn't want to make this change across all the unit test expectations unless you like this idea,